graphql_compiler/tests/test_helpers.py

```diff
@@ -18,6 +18,138 @@
 # flag to indicate a test component should be skipped
 SKIP_TEST = 'SKIP'
 
+# String of a GraphQL schema generated from OrientDB. Used for future reference testing.
+ORIENTDB_GENERATED_SCHEMA = '''
+       schema {
```

Comment:
de-indent this, and let's make sure that the auto-generated schema and the hand-written one are equivalent